core/templates/components/question-directives/question-player/question-player.component.ts

```diff
@@ -219,17 +219,17 @@ angular.module('oppia').component('questionPlayer', {
       };
 
       ctrl.getWorstSkillId = function() {
-        var minScore = Number.MAX_VALUE;
-        var worstSkillId = '';
+        var minScore = 0.95;
+        var worstSkillId = [];
         Object.keys(ctrl.scorePerSkillMapping).forEach(function(skillId) {
           var skillScoreData = ctrl.scorePerSkillMapping[skillId];
           var scorePercentage = skillScoreData.score / skillScoreData.total;
           if (scorePercentage < minScore) {
             minScore = scorePercentage;
-            worstSkillId = skillId;
+            worstSkillId.push(skillId);
           }
         });
-        return worstSkillId;
+        return worstSkillId.sort();
```

Comment:
This function isn't actually returning what you're expecting. You're setting `minScore` within the loop -- so, if the order of the skills were to change, your result would be different. 

Example: `scores = [0.05, 0.25, 0.75]` results in `worstSkillIds = [id(0.05)]` while `scores = [0.75, 0.25, 0.05]` results in `worstSkillIds = [id(0.75), id(0.25), id(0.05)]`.

Is this the behavior we want? Let me know if I missed something!

Reply:
Done. PTAL!